core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedClusterMessage.scala

```diff
@@ -95,8 +95,13 @@ private[spark] object CoarseGrainedClusterMessages {
   case class RemoveExecutor(executorId: String, reason: ExecutorLossReason)
     extends CoarseGrainedClusterMessage
 
-  case class DecommissionExecutor(executorId: String, decommissionInfo: ExecutorDecommissionInfo)
-    extends CoarseGrainedClusterMessage
+  // A message that sent from executor to driver to tell driver that the executor has been
+  // used. It's used for the case where decommission is triggered at executor (e.g., K8S)
+  case class ExecutorDecommissioned(executorId: String)
+
+  // A message that sent from driver to executor to decommission that executor.
+  // It's used for Standalone's case yet, where decommission is triggered at Worker.
```

Comment:
remove 'yet'. Are you sure that this is when the decommission is triggered at the Worker ? I think it can also be used when the decommissioning is triggered at the MasterWebUI (not necessarily a SIGPWR at the Worker).

Reply:
Ok, I actually mean it used for both two cases as they would trigger decommission at Worker side anyway. I'll update the comment to make it clearer.